lib/rethinkdb_ecto/repo.ex

```diff
@@ -72,6 +74,30 @@ defmodule RethinkDB.Ecto.Repo do
     end
   end
 
+  def convert_date(item) do
+    {attr, value} = item
+    date_string = to_date_string(value)
+
+    cond do
```

Comment:
I'm not a big `cond` fan.

I think a single pattern match would suffice here. `case value do`